buildozer/README.md

```diff
@@ -15,7 +15,7 @@ go install github.com/bazelbuild/buildtools/buildozer@latest
 ## Usage
 
 ```shell
-buildozer [OPTIONS] ['command args' | -f FILE ] label-list
+buildozer [OPTIONS] ['command args'... | -f FILE ] label-list
```

Comment:
Did you mean `'command args'...`  or   `'command args' ...`

Reply:
I want to "match" `'command args'`, `'command args' 'command args'`, .... Not entirely sure what the typical conventions for help texts are, what would you say?

Reply:
https://stackoverflow.com/a/9727046

The PR looks good to me